File: examples/simp/Python/predictor/predictor1.py
# Import the library
import LinkPredPython as lpp
# Create a predictor object
p = lpp.Predictor();
# Load network from file
p.loadnet("Zakarays_Karate_Club.edges");
# Predict the score of all non-exisitng edges using Adamic Adar index
esv = p.predAllADA();
# Print the scores
for es in esv:
	print(es.i + "\t" + es.j + "\t" + "{:.4f}".format(es.score));

File: examples/simp/CPP/predictor/predictor1.cpp
#include <linkpred.hpp>
#include <iostream>

using namespace LinkPred::Simp;

int main() {
	Predictor p; // Create a prtedictor object
	p.loadnet("Zakarays_Karate_Club.edges"); // Load network from file
	std::vector<EdgeScore> esv = p.predAllADA(); // Predict the score of all non-exisitng edges using Adamic Adar index.
	for (auto it = esv.begin(); it != esv.end(); ++it) {
		// Print the scores
		std::cout << it->i << "\t" << it->j << "\t" << it->score << std::endl;
	}
	return 0;
}
